cvc.js

```diff
@@ -94,7 +98,8 @@ class State {
 
 
     return {
-      rows: Object.values(rows),
+      // Stable sort by District, then CVC to keep the rows consistent and not move around during refreshes
+      rows: Object.values(rows).sort((a, b) => `${a.District}-${a.CVC}`.localeCompare(`${b.District}-${b.CVC}`)),
```

Comment:
There are cases where CVCs can get added/removed + times when the server does not give any data back. Would that affect consistency of the rows ?

Reply:
Yeah, it will -  I think this needs to be refactored to produce a 'diff' that takes into account current state of spreadsheet. Maybe in another PR tho?